src/screens/OtherProfile/index.js

```diff
@@ -406,12 +407,16 @@ const OtherProfile = () => {
     return (
       <>
         <View style={styles.headerImageContainer}>
-          <Image
-            style={styles.profileImage}
-            source={{
-              uri: dataMain.profile_pic_path ?? DEFAULT_PROFILE_PIC_PATH
-            }}
-          />
+          <View style={{marginRight: 22}}>
+            <CircleGradient fill={dataMain.karma_score} size={100} width={3}>
+              <Image
+                style={styles.profileImage}
+                source={{
+                  uri: dataMain.profile_pic_path ?? DEFAULT_PROFILE_PIC_PATH
+                }}
+              />
+            </CircleGradient>
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `CircleGradient` component around the profile image seems to be correct. However, it's important to ensure that the `karma_score` passed to the `fill` prop is always a valid number. If there's a chance that `dataMain.karma_score` could be `null` or `undefined`, this could cause issues. Consider adding a default value or some error handling here.
```diff
- <CircleGradient fill={dataMain.karma_score} size={100} width={3}>
+ <CircleGradient fill={dataMain.karma_score ?? 0} size={100} width={3}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
mas @yusufputra ini coba diperhatikan ya, soalnya kita di file ini pake js, aku takut miss